components/NavBar/NavBar.tsx

```diff
@@ -105,58 +97,29 @@ export const NavBar = () => {
   }
 
   return (
-    <>
-      <UserNavbarContainer>
-        <UserNavbar aria-label={intl.formatMessage({ id: "navbar.ariaUserMenu" })}>
-          <Cart />
-          {/* Hiding unused icons
-          <Button href="/tasks" variant="ghost" color="secondary">
-            <FormattedMessage id="navbar.taskStatistic" values={{ remaining: <span>0</span> }} />
-          </Button>
-          <Button variant="ghost_icon" color="secondary">
-            <Icon data={refresh} title={intl.formatMessage({ id: "navbar.titleRefresh" })} />
-          </Button>  */}
-          <Button variant="ghost_icon" color="secondary">
-            <Icon data={account_circle} title={intl.formatMessage({ id: "navbar.titleAccount" })} />
-          </Button>
-        </UserNavbar>
-      </UserNavbarContainer>
-
-      <Header>
-        <HeaderContentContainer>
-          <LogoContainer>
-            <NextLink href="/" passHref>
-              <Button as="a" variant="ghost" color="secondary">
-                <Typography><FormattedMessage id="navbar.logo" /></Typography>
-              </Button>
-            </NextLink>
-          </LogoContainer>
-
-          <nav aria-label="Main navigation">
-            <ActionsContainer>
-              <Link href="/browse">
-                <Button variant="ghost" color="secondary">
-                  <Icon data={explore} title={intl.formatMessage({ id: "navbar.browse" })} />
-                  <FormattedMessage id="navbar.browse" />
-                </Button>
-              </Link>
-              {/*
-              <Link href="/browse">
-                <Button variant="ghost" color="secondary">
-                  <Icon data={comment_discussion}
-                   title={intl.formatMessage({ id: "navbar.community" })} />
-                  <FormattedMessage id="navbar.community" />
-                </Button>
-              </Link>
-              */}
-            </ActionsContainer>
-          </nav>
-
+    <HeaderContentContainer>
```

Comment:
You need to have the content container inside of the Topbar, since the Topbar is not sticky with respect to the HeaderContentContainer (the parent element). Since the HeaderContentContainer is the same height as the Top bar itself, you will not have the sticky effect. Try to swap them around and see if that doesn't help you with the sticky feature :)